### PR TITLE
Add rabbitmq

### PR DESCRIPTION
**Pull request by dreamer-nitj**
Add the rabbitmq hardened image to community images

This PR adds the coverage script, README, functional test for rabbitmq image.

Test Plan:
[x] Ran the run.sh successfully
[x] Ran the functional.sh with success
[x] Generated the README file

### CODE REVIEW

**Review by codervinod — APPROVED** [review]

**Comment by codervinod** [comment]
Please add rabbit-mq perf test

**Comment by dreamer-nitj** [comment]
> Please add rabbit-mq perf test

done

**Review by codervinod — APPROVED** [review]

**Review by codervinod — CHANGES_REQUESTED** [review]
kubectl run -it perf-test ; remove -it as tests are failing

File: community_images/rabbitmq/bitnami/coverage.sh [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    # delete the generated command files
+    rm "$SCRIPTPATH"/publish_commands.sh
+    rm "$SCRIPTPATH"/consume_commands.sh
+
+    PERF_POD="perf-test"
+    DEFAULT_RABBITMQ_USER='user'
+    PERF_TEST_IMAGE_VERSION='2.18.0'
+
+    # run the perf benchmark test
+    kubectl run -it "${PERF_POD}" \
```
Comment (codervinod) [inline community_images/rabbitmq/bitnami/coverage.sh:55]: remove -it as github actions it non-tty. We need to use kubectl run -i. Remove "t"

Comment (dreamer-nitj) [reply community_images/rabbitmq/bitnami/coverage.sh]: done

**Review by codervinod — APPROVED** [review]

**Pull request merged**